classes/form/CustomerFormatter.php

```diff
@@ -27,15 +27,15 @@
 
 class CustomerFormatterCore implements FormFormatterInterface
 {
-    private $translator;
-    private $language;
+    protected $translator;
```

Comment:
this is not something we want to open, you can alter it by injecting your own in the constructor

Reply:
@mickaelandrieu I don't want to alter it, i just want to use it... And if i wanted to alter it, what is the problem?